DESCRIPTION.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
The [OSC governance policy](https://intersect.gitbook.io/open-source-committee/policies/governance/documentation) expects a DESCRIPTION file. 
However, this is redundant as all information is already included in the README file.
Also, the [OSC Documentation Repo](https://github.com/IntersectMBO/OSC-documentation) itself doesn't have a DESCRIPTION file.
I'm unsure what to do here. Is the DESCRIPTION file really necessary?